### PR TITLE
data:MahekUnnisa

### PR DESCRIPTION
**Pull request by MahekUnnisa**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3432"></a>

### CODE REVIEW

File: data/MahekUnnisa.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/MahekUnnisa",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Hashnode",
+      "url": "https:mahekunnisa.hashnode.dev",
+      "icon": "FaHashnode"
```
Comment (Aadarsh805) [inline data/MahekUnnisa.json:26]: use `SiHashnode` this for the Hashnode icon

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/MahekUnnisa.json (hunk 1 above)
Comment (MahekUnnisa) [reply data/MahekUnnisa.json]: Thanks for letting me know @Aadarsh805 . I have changed the file accordingly.

File: data/MahekUnnisa.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/MahekUnnisa",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Hashnode",
+      "url": "https:mahekunnisa.hashnode.dev",
```
Comment (EmmaDawsonDev) [inline data/MahekUnnisa.json:25]:
```suggestion
      "url": "https://mahekunnisa.hashnode.dev",
```

**Review by EmmaDawsonDev — DISMISSED** [review]
There's a small error in the hashnode URL. Once that's updated then this is good to be merged.

File: data/MahekUnnisa.json (hunk 2 above)
Comment (MahekUnnisa) [reply data/MahekUnnisa.json]: Thanks for reviewing. I have changed it.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**